MvvmCross/Platform/Uap/Views/MvxBaseApplication.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Threading.Tasks;
+using MvvmCross.Platform.Uap.Core;
+using MvvmCross.Platform.Uap.Views.Suspension;
+using MvvmCross.ViewModels;
+using Windows.ApplicationModel;
+using Windows.ApplicationModel.Activation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+namespace MvvmCross.Platform.Uap.Views
+{
+    public abstract class MvxBaseApplication: Application
```

Comment:
I would remove the word `Base` here. We just put Mvx in front of every class throughout the framework.

Reply:
ok